baxter_collaboration_lib/include/robot_interface/robot_interface.h

```diff
@@ -571,9 +571,10 @@ class RobotInterface
     std::string  getLimb() { return  _limb; };
     State       getState() { return _state; };
 
-    geometry_msgs::Point        getPos()    { return    _curr_pos; };
-    geometry_msgs::Quaternion   getOri()    { return    _curr_ori; };
-    geometry_msgs::Wrench       getWrench() { return _curr_wrench; };
+    geometry_msgs::Point        getPos()         { return    _curr_pos; };
+    geometry_msgs::Quaternion   getOri()         { return    _curr_ori; };
+    geometry_msgs::Wrench       getWrench()      { return _curr_wrench; };
```

Comment:
Why did you have to change these three lines? Did you add some tabs/spaces that are not needed?